RestartManager.js

```diff
@@ -0,0 +1,33 @@
+let NativeCodePush = require("react-native").NativeModules.CodePush;
+
+const RestartManager = (() => {
+    let _allowed = true;
+
+    function restartApp(onlyIfUpdateIsPending = false) {
```

Comment:
If the restart isn't currently allowed, I wonder if we should indicate that to the caller somehow. Maybe it isn't too important for now, but it's something to keep in mind.